app/assets/javascripts/services/teamSvc.js

```diff
@@ -19,6 +19,14 @@ angular.module('QuepidApp')
         this.members      = members;
         this.scorers      = scorers;
         this.owned        = owned;
+
+
+        angular.forEach(this.cases, function(c) {
+          // This is really ugly.
```

Comment:
Revisit this code if https://github.com/o19s/quepid/issues/65 sets up lazy load for cases within a team.